Sources/StreamVideo/OpenApi/generated/Models/HealthCheckEvent.swift

```diff
@@ -1,39 +1,30 @@
 //
-// HealthCheckEvent.swift
-//
-// Generated by openapi-generator
-// https://openapi-generator.tech
+// Copyright © 2024 Stream.io Inc. All rights reserved.
 //
 
 import Foundation
-
-
-public struct HealthCheckEvent: @unchecked Sendable, Event, Codable, JSONEncodable, Hashable, WSClientEvent {
-    /** The connection_id for this client */
+    
+public struct HealthCheckEvent: @unchecked Sendable, Event, Codable, JSONEncodable, Hashable {
+    
+    public var cid: String
```

Comment:
are we sure about this one? `cid` is chat thing and I don't see it as mandatory.

Reply:
It is bringing the chat event because they have the same type `health.check`.

Reply:
This breaks the parsing, here's the current response of the video health check:
```
▿ 4 elements
  ▿ 0 : 2 elements
    - key : custom
    - value : 0 elements
  ▿ 1 : 2 elements
    - key : type
    - value : health.check
  ▿ 2 : 2 elements
    - key : connection_id
    - value : 66d72610-0a05-4c41-0200-0000001aa896
  ▿ 3 : 2 elements
    - key : created_at
    - value : 2024-09-04T11:16:46.693702464Z
```